src/frontend/AppContainerWrapper.tsx

```diff
@@ -138,9 +175,8 @@ const AppContainerWrapper = () => {
         ? {}
         : {
             loadNavigationState: async () => {
-              const loadedNavState = await loadSavedNavState(
-                experiments.onboarding
-              );
+              const loadedNavState =
+                (await loadSavedNavState(experiments.onboarding)) || undefined;
```

Comment:
would lean towards suggestion in https://github.com/digidem/mapeo-mobile/pull/923/files#r806082136 instead of the fallback here